torch/csrc/autograd/init.cpp

```diff
@@ -616,8 +627,8 @@ static PyMethodDef methods[] = { // NOLINT
   {"is_anomaly_enabled", is_anomaly_mode_enabled, METH_NOARGS, nullptr},
   {"_enter_dual_level", python_enter_dual_level, METH_NOARGS, nullptr},
   {"_exit_dual_level", castPyCFunctionWithKeywords(python_exit_dual_level), METH_VARARGS | METH_KEYWORDS, nullptr},
-  {"_enter_python_mode", enter_python_mode, METH_O, nullptr},
-  {"_exit_python_mode", exit_python_mode, METH_NOARGS, nullptr},
+  {"_set_python_mode", set_python_mode, METH_O, nullptr},
+  {"_get_python_mode", get_python_mode, METH_NOARGS, nullptr},
```

Comment:
nit: we probably want to start referring to these as set_torch_dispatch_mode / get_torch_dispatch_mode to differentiate it from torch_function mode. "python mode" by itself is ambiguous. Doesn't need to happen in this PR